functions.php

```diff
@@ -1319,3 +1340,271 @@ function wp_autoupdates_themes_bulk_actions_handle( $redirect_to, $doaction, $it
+					unset( $wp_autoupdate_plugins[ $index ] );
+					update_site_option( 'wp_auto_update_plugins', $wp_autoupdate_plugins );
+
+					/** This filter is documented in wp-admin/includes/class-wp-plugins-list-table.php */
+					$all_plugins           = apply_filters( 'all_plugins', get_plugins() );
+					$wp_autoupdate_plugins = array_intersect( $wp_autoupdate_plugins, array_keys( $all_plugins ) );
+					$enabled_count         = count( $wp_autoupdate_plugins );
+					$plugin_name           = $all_plugins[ $asset ]['Name'];
+
+					$plugin_url = add_query_arg(
+						array(
+							'action'   => 'autoupdate',
+							'plugin'   => $asset,
+							'_wpnonce' => wp_create_nonce( 'autoupdate-plugin_' . $asset ),
+						),
+						'plugins.php'
+					);
+
+					$return_html = sprintf(
+						'<p class="plugin-autoupdate-disabled"><a href="%s" class="edit plugin-autoupdate-enable" aria-label="%s"><span class="plugin-autoupdate-label">%s</span></a></p>',
+						esc_url_raw( $plugin_url ),
+						esc_html__( 'Auto-updates enabled', 'wp-autoupdate' ),
+						esc_html__( 'Enable auto-updates', 'wp-autoupdates' )
+					);
+
+					wp_send_json_success(
+						array(
+							'enabled_count'  => '(' . $enabled_count . ')',
+							'disabled_count' => '(' . absint( count( $all_plugins ) - $enabled_count ) . ')',
+							'return_html'    => $return_html,
+							'type'           => 'plugin',
+						)
+					);
+					break;
+				}
+			}
+		}
+	} elseif ( 'theme' === $type ) {
+		$wp_autoupdate_themes = get_site_option( 'wp_auto_update_themes', array() );
+		if ( is_array( $wp_autoupdate_themes ) && ! empty( $wp_autoupdate_themes ) ) {
+			foreach ( $wp_autoupdate_themes as $index => $theme_file ) {
+				if ( $theme_file === $asset ) {
+					unset( $wp_autoupdate_themes[ $index ] );
+					update_site_option( 'wp_auto_update_themes', $wp_autoupdate_themes );
+
+					/** This filter is documented in wp-admin/includes/class-wp-plugins-list-table.php */
+					$all_themes           = wp_get_themes();
+					$wp_autoupdate_themes = array_intersect( $wp_autoupdate_themes, array_keys( $all_themes ) );
+					$enabled_count        = count( $wp_autoupdate_themes );
+					$theme_name           = $all_themes[ $asset ]->get( 'Name' );
+
+					$theme_url = add_query_arg(
+						array(
+							'action'   => 'autoupdate',
+							'theme'    => $asset,
+							'_wpnonce' => wp_create_nonce( 'autoupdate-theme_' . $asset ),
+						),
+						'themes.php'
+					);
+
+					$return_html = sprintf(
+						'<p class="theme-autoupdate-disabled"><a href="%s" class="edit theme-autoupdate-enable" aria-label="%s"><span class="theme-autoupdate-label">%s</span></a></p>',
+						esc_url_raw( $theme_url ),
+						esc_html__( 'Enable automatic updates for', 'wp-autoupdate' ) . ' ' . esc_html( $theme_name ),
+						esc_html__( 'Enable auto-updates', 'wp-autoupdates' )
+					);
+
+					wp_send_json_success(
+						array(
+							'enabled_count'  => '(' . $enabled_count . ')',
```

Comment:
Can you add an `absint` here?
```suggestion
							'enabled_count'  => '(' . absint( $enabled_count ) . ')',
```